### PR TITLE
Add shorthand search terms 'Uni' and 'Hbf'

### LINKED ISSUE #1337: 'uni' doesn't work as shorthand for 'university' in search
Searching for example "Uni of Hertfordshire" doesn't return the uni in the top results, but searching for "University of Hertfordshire" does.

### PR DESCRIPTION
**Pull request by endim8**
Closes #1337

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -2172,7 +2172,7 @@ da:4Togstation|4station|3jernbane
 nl:4Station|4spoorwegen|spoor|3trein
 fi:Rautatieasema
 fr:3Gare|4chemin de fer|3train
-de:4Bahnhof|3Haltepunkt|3Station|Zug|Bahn|ÖPNV
+de:4Bahnhof|3Haltepunkt|3Station|Zug|Bahn|ÖPNV|3Hbf
```
Comment (vng) [inline data/categories.txt:2175]: 3 doesn't make sense here, since "Hbf" has 3 letters.

File: data/categories.txt [hunk 2]
```diff
@@ -4795,7 +4795,7 @@ sk:3Toalety|wc
 mr:शौचालय
 
 amenity-university
-en:4University|U+1F393
+en:4University|U+1F393|2Uni
```
Comment (vng) [inline data/categories.txt:4798]: 2 means that after entering "Un", the app will show "Uni" as _first suggest_ result. Not good, imho, Uni is very specific term.

Can leave Uni without 2 prefix, but I have doubts about it.

Comment (biodranik) [reply data/categories.txt:4798]: Uni is a commonly used acronym. I agree that digits should be removed.

File: data/categories.txt (hunk 1 above)
Comment (endim8) [reply data/categories.txt]: removed 👍

**Review by vng — APPROVED** [review]

**Pull request merged**